public/coffee/ide/directives/layout.coffee

```diff
@@ -103,12 +103,12 @@ define [
 
 					resetOpenStates()
 					onInternalResize()
-					
-					scope.$watch attrs.layoutDisabled, (value) ->
-						if value
-							element.layout().hide("east")
-						else
-							element.layout().show("east")
-							
+
+					if attrs.layoutDisabled?
```

Comment:
There is no chance this is updated dynamically at some point, but we'll miss it because we didn't watch it at the beginning?